.github: Add some templates

Jay-Jay-OPL opened the pull request:
- pickup from https://github.com/a2o/snoopy/tree/master/.github
- Thanks a lot @bostjan :)

Signed-off-by: Caio Oliveira caiooliveirafarias0@gmail.com

Caio99BR committed:
.github: Add some templates

* pickup from https://github.com/a2o/snoopy/tree/master/.github
* Thanks a lot @bostjan :)

Signed-off-by: Caio Oliveira <caiooliveirafarias0@gmail.com>

Caio99BR commented:
Do it: http://p.sc2.nl/ry3BPK5yx

To find changed files use `gedit $(git diff --name-only | uniq)` or `git gui`

Caio99BR committed:
OPL: Last commit about revision, i hate it now

TravisCI:
* Use file_glob to pickup files

Makefile:
* Tested and Fix REVISION
* Add packing of zip to RELEASE builds

lng_pack.sh:
* Rework pickup of REVISION

Signed-off-by: Caio Oliveira <caiooliveirafarias0@gmail.com>

Jay-Jay-OPL commented:
@Caio99BR, thanks, I appreciate the instructions.  But I think I might have to do that each time for not just that file, but for others.  As soon as you the icaro repo updates their Lang Files, that's when my repo begins to drift, since my repo has a few new features that require two new lines in the lang files.

I saw the screenshot, but I didn't quite understand it.  There is a lot of code you did that I don't understand those commands.

When you find the time, can you do a step by step on what you did, plus an explanation on what each command does, this way I can learn and understand what I am doing with each sequence in those commands.  

Thanks again for your help.  It's highly appreciated! :)

BatRastard committed:
Per Game CHEATS should be good now ... (#14)

* 1st Attempt - Per Game CHEAT

Name says it all. Haven't compiled nor tested this myself, but
considering I well into my 6 pack of Coronas, shit's probably a tad
broken! Got laundry to do now. Alas, the laundry room is a good stagger
or three around the corner ... =}

* Whoops!

*belch*

* Attempt #2 Per Game CHEAT

Fixing a few things ...

* Attempt #3 - More syntax fixes ...

I'll sober up here in a moment ...

* Hooboy!

Getting there. Learning Git, too! Aaaand missing Mercurial!

* Ugh ...

Bastard ...

* Damn

Last try before bed ...

* Piss on this shit ...

I can't figure out the problem with "supportbase.c" telling me my
defines aren't defined and there's unterminated #ifdefs.
Bullfuckingshit.

* OPL: Fix errors introduced by Per-Game Cheat

Signed-off-by: Caio Oliveira <caiooliveirafarias0@gmail.com>

Commit:
Per-Game Cheat Engine fix

Commit:
Per-Game Cheat Engine fix

Commit:
Revert "Per-Game Cheat Engine fix"

This reverts commit 66150f5f58bdd958ba1bccea324290130bcc3cc0.

Commit:
Per-Game Cheat Engine fix finished. Phew!

ElPatas1 committed:
Update languages

ElPatas1 committed:
- SP193: Change in the PS2SDK about libpwroff. In one commit last year, i changed its design
(along with AUDSRV) to use SIFRPC instead because of the change in design of the user
SIFCMD buffer. Unfortunately, i didn't add proper re-initialization code to the new design of
libpwroff. So when OPL starts the HDLDump server the old libpwroff SIFRPC server is contacted
and the EE crashes.
Server was broken in r878, and this change was made in 09-27-2016, working well again in r930.
- Adjusted the HDLDump server thread priorities for proper functionality of the
transfer data.

Commit:
Per-Game Cheat Engine - fix for ETH and HDD Modes

Caio99BR commented:
@Jay-Jay-OPL, you can make a new lng/ folder and change lng_pack
Also you can use this https://github.com/Caio99BR/FalconSSKernel/blob/rel/patch.sh for apply patchs and keep original author (if you want)
